app/Models/Entry.php

```diff
@@ -222,7 +222,7 @@ public function content(bool $withEnclosures = true, bool $allowDuplicateEnclosu
 				$content .= '<p class="enclosure-credits">© ' . $credit . '</p>';
 			}
 			if ($description != '') {
-				$content .= '<figcaption class="enclosure-description">' . $description . '</figcaption>';
+				$content .= '<figcaption>' . $description . '</figcaption>';
```

Comment:
```suggestion
				$content .= '<figcaption class="enclosure-description">' . $description . '</figcaption>';
```

Let's keep the class, which can be used also for semantic reasons